build/azure-pipelines/build-product.yml

```diff
@@ -80,63 +80,47 @@ stages:
         inputs:
           buildType: 'current'
           targetPath: '$(Pipeline.Workspace)'
-      - task: UseDotNet@2
-        inputs:
-          packageType: 'sdk'
-          version: '2.1.x'      
-      - task: EsrpCodeSigning@1
+      - task: EsrpCodeSigning@2
         displayName: Sign Windows binary
         inputs:
           ConnectedServiceName: 'Code Signing'
           FolderPath: '$(Pipeline.Workspace)'
           Pattern: 'sqlcmd.exe'
           signConfigType: 'inlineSignParams'
-          SessionTimeout: '600'
-          MaxConcurrency: '5'
-          MaxRetryAttempts: '5'
           inlineOperation: |
             [
               {
                 "keyCode": "CP-230012",
                 "operationSetCode": "SigntoolSign",
                 "parameters": [
-                {
-                  "parameterName": "OpusName",
-                  "parameterValue": "go-sqlcmd"
-                },
-                {
-                  "parameterName": "OpusInfo",
-                  "parameterValue": "https://github.com/microsoft/go-sqlcmd"
-                },
-                {
-                  "parameterName": "PageHash",
-                  "parameterValue": "/NPH"
-                },
-                {
-                  "parameterName": "FileDigest",
-                  "parameterValue": "/fd sha256"
-                },
-                {
-                  "parameterName": "TimeStamp",
-                  "parameterValue": "/tr \"http://rfc3161.gtm.corp.microsoft.com/TSS/HttpTspServer\" /td sha256"
-                }
+                  {
+                    "parameterName": "OpusName",
+                    "parameterValue": "go-sqlcmd"
+                  },
+                  {
+                    "parameterName": "OpusInfo",
+                    "parameterValue": "https://github.com/microsoft/go-sqlcmd"
+                  },
+                  {
+                    "parameterName": "PageHash",
+                    "parameterValue": "/NPH"
+                  },
+                  {
+                    "parameterName": "FileDigest",
+                    "parameterValue": "/fd sha256"
+                  },
+                  {
+                    "parameterName": "TimeStamp",
+                    "parameterValue": "/tr \"http://rfc3161.gtm.corp.microsoft.com/TSS/HttpTspServer\" /td sha256"
+                  }
                 ],
                 "toolName": "signtool.exe",
                 "toolVersion": "6.2.9304.0"
-              },
-              {
-                "keyCode": "CP-230012",
-                "operationSetCode": "SigntoolVerify",
-                "parameters": [
-                {
```

Comment:
@shueybubbles :This piece of code was not auto generated. Should I retain this "SigntoolVerify" block as well?

Reply:
Retained this code block. Verified the run [here](https://mssqltools.visualstudio.com/CrossPlatBuildScripts/_build/results?buildId=186071&view=logs&j=2524592f-1200-5da2-277a-7cff66fe62fe&t=0771f747-4673-5e21-eb1d-b95b586710a2)